tools/xtce_generation/telemetry_generator.py

```diff
@@ -250,9 +250,19 @@ def create_remaining_parameters(self, parameter_set):
 
             description = Et.SubElement(parameter, "xtce:LongDescription")
 
-            description.text = (
-                row.get("shortDescription") or row.get("longDescription") or ""
+            # handle nan floating point values and replace with empty strings
+            short_description = (
+                ""
+                if str(row.get("shortDescription")) == "nan"
+                else row.get("shortDescription")
             )
+            long_description = (
+                ""
+                if str(row.get("longDescription")) == "nan"
+                else row.get("longDescription")
+            )
+
+            description.text = short_description or long_description or ""
```

Comment:
With above lines, do we need that last `or`?
```suggestion
            description.text = short_description or long_description
```

Reply:
Yeah I think so. What I added just handles the case I was seeing where I had `nan` values. It seems from how the code is written that some people were getting `None` values which is what that `or ""` should handle. I don't know if I had `None` anywhere, but I'm assuming others did.

Reply:
Ah I see. Then let's keep it.